generator-mediator-js/app/prompts.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+var config = require('./config')
+
+exports.promptsList = (enablePrompts) => {
+  var list = [
+    {
+      when: function() { return enablePrompts },
+      type: 'confirm',
+      name: 'enablePrompts',
+      message: 'Would you like to be prompted for config?',
+      default: config.enablePrompts
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'mediatorName',
+      message: 'What is your Mediator\'s name?',
+      default: config.mediatorName
+    }, {
+      when: function(response) { return response.enablePrompts },
+      type: 'input', 
+      name: 'mediatorDesc', 
+      message: 'What does your Mediator do?',
+      default: config.mediatorDesc
```

Comment:
I'm not sure we should include a default description. A default description will never be right. The same goes for name above.